src/vendors/oasisscan.ts

```diff
@@ -108,3 +126,26 @@ export function parseTransactionsList(transactionsList: OperationsRow[]): Transa
     return parsed
   })
 }
+
+export function parseDelegations(delegations: DelegationRow[]): Delegation[] {
+  return delegations.map(delegation => {
+    const parsed: Delegation = {
+      amount: parseStringValueToInt(delegation.amount).toString(),
+      shares: parseStringValueToInt(delegation.shares).toString(),
+      validatorAddress: delegation.validator_address,
+    }
+    return parsed
+  })
+}
+export function parseDebonding(debonding: DebondingDelegationRow[]): DebondingDelegation[] {
+  return debonding.map(debonding => {
+    const parsed: DebondingDelegation = {
+      // TODO: use amount field when it is available
+      amount: (parseStringValueToInt(debonding.shares) * 1).toString(),
```

Comment:
`parseStringValueToInt(debonding.shares) * 1).toString()` 
we multiple a string value to make it a number and then we cast it to string again. Do I understand it correctly?

Reply:
the `1` I think is a placeholder for when we get proper debonding pool share price information in the future

Reply:
Made it more explicit to avoid confusion :)
```ts
    // TODO: use amount field, or share price when it is available. Until then,
    // using price=1 is inaccurate if debonding pool gets slashed.
    const sharePrice = 1
    const parsed: DebondingDelegation = {
      amount: (parseStringValueToInt(debonding.shares) * sharePrice).toString(),
```